doc/DataFormats.md

```diff
@@ -54,8 +54,8 @@ This message type is initiated by the EthBridge, as part of the creation of a ne
 
 Type-specific data:
 
-* challenge period, in L1 blocks (uint)
-* ArbGas speed limit, in ArbGas per L1 block (uint)
+* challenge period, in seconds (uint)
+* ArbGas speed limit, in ArbGas per second (uint)
```

Comment:
At the moment the "challenge period, in second" parameter isn't used for anything in ArbOS.  We could make it be measured in blocks, but my guess was that seconds would more likely be the unit at whatever future point this might start getting used.